docs/general/dns-providers.md

```diff
@@ -821,6 +821,17 @@ These servers block adult websites and inappropriate contents.
 | DNS-over-HTTPS | `https://public.dns.iij.jp/dns-query` | [Add to AdGuard](adguard:add_dns_server?address=https://public.dns.iij.jp/dns-query&name=public.dns.iij.jp), [Add to AdGuard VPN](adguardvpn:add_dns_server?address=https://public.dns.iij.jp/dns-query&name=public.dns.iij.jp) |
 | DNS-over-TLS | `tls://public.dns.iij.jp` | [Add to AdGuard](adguard:add_dns_server?address=tls://public.dns.iij.jp&name=public.dns.iij.jp), [Add to AdGuard VPN](adguardvpn:add_dns_server?address=tls://public.dns.iij.jp&name=public.dns.iij.jp) |
 
+### JupitrDNS
+
+[JupitrDNS](https://jupitrdns.com/) is a a free hobby DNS service that blocks ads, trackers and malware.
```

Comment:
Not sure if we need to mention "hobby" here, but maybe it's useful to include that it's a recursive, no logs, DNSSEC Support, no EDNS Client Subnet DNS server

Reply:
also comma before "and"